bat/tests/09-ister-config/run.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+@test "Build image using default ister config" {
+  mixer $MIXARGS init --clear-version $CLRVER --mix-version 10
+  mixer $MIXARGS bundle add editors
+
+  # `build image` should fail because the prerequisite commands are not executed.
+  # This is intentional in order to reduce test execution time.
+  # The goal is to unit test the creation of ister config file with relevant bundles.
+  run sudo mixer $MIXARGS build image --native
+  [[ "$status" -eq 1 ]]
+  [[ "$output" =~ "release-image-config.json not found" ]]
+  [[ "$output" =~ "Copying image template" ]]
+  [[ "$output" =~ "Updating image bundle list based on mixbundles" ]]
+}
+
+@test "Update ister config file with mix bundle list" {
+  run cat release-image-config.json
+  [[ "$status" -eq 0 ]]
+  [[ "$output" =~ os-core\" ]]
```

Comment:
is `\"` really needed in the regex?

Reply:
Yes. This is done in-order to differentiate between os-core and os-core-update.

Reply:
Okay, then I would prefer the regex to be `\"os-core\"`

Reply:
Fair enough. In fact, I'm taking it a step further and doing it for all the bundles.